doc/tor.md

```diff
@@ -180,7 +195,7 @@ should be equal to binding address and port for inbound Tor connections (127.0.0
 
 In a typical situation, where you're only reachable via Tor, this should suffice:
 
-	./dashd -proxy=127.0.0.1:9050 -externalip=ssapp53tmftyjmjb.onion -listen
+	./dashd -proxy=127.0.0.1:9050 -externalip=7zvj7a2imdgkdbg4f2dryd5rgtrn7upivr5eeij4cicjh65pooxeshid.onion -listen
```

Comment:
19961: should use one of known tor relays listed below

Reply:
Wouldn't it be more desirable for the `externalip` value to not conflict with an actual Tor relay?

Reply:
good point, pls ignore this one then